src/controllers/contactController.ts

```diff
@@ -0,0 +1,59 @@
+const ContactModel = require('../models/contact');
+
+export const displayContacts = async (req: any, res: any) => {
```

Comment:
Don't forget to use types ... This is why we use TypeScript 😆 https://github.com/ThomasF34/api-admitech/blob/init/spec/src/routes/offre.ts

Reply:
Got it!